packages/@okta/vuepress-site/code/react/okta_react_sign-in_widget/index.md

```diff
@@ -68,13 +68,16 @@ Create a `src/OktaSignInWidget.js` file:
 
 ```jsx
 import React, { Component } from 'react';
-import ReactDOM from 'react-dom';
```

Comment:
We should also update this page to say "this example uses okta-react vX.Y.Z, react 16.7.x, and react-router 5.x" (with whatever the actual version numbers are)